src/views/credentials/addCredentialModal.tsx

```diff
@@ -270,25 +342,43 @@ const CredentialFormFields: React.FC<CredentialFormFieldsProps> = ({
             name="become_password"
             onChange={event => handleInputChange('become_password', (event.target as HTMLInputElement).value)}
             ouiaId="become_password"
+            validated={errors?.become_password ? 'error' : 'default'}
           />
+          {ErrorFragment(errors?.become_password)}
         </FormGroup>
       </React.Fragment>
     )}
   </React.Fragment>
 );
+
+const ErrorFragment = (errorMessage: string | undefined) => {
+  return (
+    <FormHelperText>
+      <HelperText>
+        <HelperTextItem
+          variant={errorMessage ? 'error' : 'default'}
+          {...(errorMessage && { icon: <ExclamationCircleIcon /> })}
+        >
+          {errorMessage}
+        </HelperTextItem>
+      </HelperText>
+    </FormHelperText>
+  );
+};
```

Comment:
Is there a better way to create a component from a function? the way this one was created I had to invoke it as `{ErrorFragment(errorMessage)}` instead of `<ErrorFragment errorMessage="some error">`